src/coreclr/jit/lowerxarch.cpp

```diff
@@ -5170,7 +5170,7 @@ void Lowering::ContainCheckSIMD(GenTreeSIMD* simdNode)
 // Return Value:
 //    true if 'node' is a containable hardware intrinsic node; otherwise, false.
 //
-bool Lowering::IsContainableHWIntrinsicOp(GenTreeHWIntrinsic* containingNode, GenTree* node, bool* supportsRegOptional)
+bool Lowering::IsContainableHWIntrinsicOp(GenTreeHWIntrinsic* containingNode, GenTree*& node, bool* supportsRegOptional)
```

Comment:
Our coding convention [forbids](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/clr-jit-coding-conventions.md#1535-reference-arguments) such use of reference parameters.

Reply:
Resolved by refactoring to `TryGetContainableHWIntrinsicOp` and taking `GenTree**`